integration_tests/deploy_check.py

```diff
@@ -103,5 +98,17 @@ def _deploy_and_test(appdir, language, is_xrt):
             deploy_app.stop_app(version)
 
 
+@retry(wait_fixed=4000, stop_max_attempt_number=8)
```

Comment:
This seems like a lot of retries, are we seeing it take this long in practice?

Reply:
I copied this from the retry logic in [test_root.py](https://github.com/GoogleCloudPlatform/runtimes-common/blob/master/integration_tests/testsuite/test_root.py#L34) in the integration_test framework, but I believe even that was fairly arbitrary. 

@nkubala probably has a better idea of how long this normally takes.

Reply:
we could probably up the wait time and decrease the number of tries, but I don't think 30 seconds is actually that egregious for App Engine deploys.